pdf_screenshot.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
```

Comment:
IMO, "screenshot" is not a very fortunate name for this script. Maybe something like `pdf2image.sh`?